web/src/pages/BookWizard/OverviewPage.tsx

```diff
@@ -31,15 +47,109 @@ export const OverviewPage: React.FC<OverviewPageProps> = ({onSubmit, modalLangua
+                <div className={styles.buttonInactive}>Learn</div>
+              </div>
+            </div>
+
+            <div className={styles.videoContainer}>
+              <iframe className={styles.video} src={readTab.video}></iframe>
+            </div>
+
+            <div className={styles.markdownText}>
+              <ReactMarkdown components={mdComponents}>
+                {readTab.body}
+              </ReactMarkdown>
+            </div>
+          </div>
+
+          <div className={styles.tabContainer}>
+            <div className={styles.imageContainer}>
+              <img src={previewUrl} alt='image' className={styles.coverImage}/>            
+            </div>
+            <div className={styles.bookTitle}>{title}</div>
+            <div className={styles.bookAuthor}>By {author}</div>
+
+            <div className={styles.buttonGroup}>
+              <div className={styles.tabButtonInactive}>
+                <div className={styles.buttonInactive}>Read</div>
+              </div>
+              <div className={styles.tabButtonActive}>
+                <div className={styles.buttonActive}>Explore</div>
+              </div>
+              <div className={styles.tabButtonInactive}>
+                <div className={styles.buttonInactive}>Learn</div>
+              </div>
+            </div>
+
+            <div className={styles.videoContainer}>
+              <iframe className={styles.video} src={exploreTab.video}></iframe>
+            </div>
+
+            <div className={styles.markdownText}>
+              <ReactMarkdown components={mdComponents}>
+                {exploreTab.body}
+              </ReactMarkdown>
+            </div>
+          </div>
+
+          <div className={styles.tabContainer}>
+            <div className={styles.imageContainer}>
+              <img src={previewUrl} alt='image' className={styles.coverImage}/>            
+            </div>
+            <div className={styles.bookTitle}>{title}</div>
+            <div className={styles.bookAuthor}>By {author}</div>
+
+            <div className={styles.buttonGroup}>
+              <div className={styles.tabButtonInactive}>
+                <div className={styles.buttonInactive}>Read</div>
+              </div>
+              <div className={styles.tabButtonInactive}>
+                <div className={styles.buttonInactive}>Explore</div>
+              </div>
+              <div className={styles.tabButtonActive}>
+                <div className={styles.buttonActive}>Learn</div>
+              </div>
+            </div>
+
+            <div className={styles.videoContainer}>
+              <iframe className={styles.video} src={learnTab.video}></iframe>
+            </div>
+
+            <div className={styles.markdownText}>
+              <ReactMarkdown components={mdComponents}>
+                {learnTab.body.indexOf("<p>") == -1 ?
+                  learnTab.body : learnTab.body.substring(3, learnTab.body.length - 4)
+                }
```

Comment:
We really shouldn't need this - since you mentioned the read tab issue went away on its own maybe you can try removing this now?